api/v1/api.go

```diff
@@ -77,18 +81,160 @@ type postIsiExportResp struct {
 type IsiExport struct {
 	ID    int      `json:"ID"`
 	Paths []string `json:"paths"`
+	Clients []string `json:"clients"`
 }
 
 type getIsiExportsResp struct {
 	ExportList []*IsiExport `json:"exports"`
 }
 
+type isiThresholds struct {
+	Advisory int64 `json:"advisory"`
+	AdvisoryExceeded bool `json:"advisory_exceeded"`
+	AdvisoryLastExceeded interface{} `json:"advisory_last_exceeded"`
+	Hard int64 `json:"hard"`
+	HardExceeded bool `json:"hard_exceeded"`
+	HardLastExceeded interface{} `json:"hard_last_exceeded"`
+	Soft int64 `json:"soft"`
+	SoftExceeded bool `json:"soft_exceeded"`
+	SoftLastExceeded interface{} `json:"soft_last_exceeded"`
+}
+
+type IsiQuota struct {
+	Container bool `json:"container"`
+	Enforced  bool `json:"enforced"`
+	Id        string `json:"id"`
```

Comment:
Changed all to variables to Id.  The `json:"id"` is now lower case to be consistent with the json returned by PAPI.